frontend/src/components/employees/index.tsx

```diff
@@ -178,8 +159,15 @@ const Employees = () => {
     <div className={employeesStyle.employees}>
       <h1 className={employeesStyle.header}> Employees </h1>
       {addButton}
-      <div className={employeesStyle.tableContainer}>
-        {renderTable(tableColumns)(staffList)}
+      <div
+        className={[employeesStyle.tableContainer, 'ag-theme-alpine'].join(
```

Comment:
Can just interpolate it

Reply:
Can you explain what you mean by 'interpolate' here?

Reply:
```
`${employeesStyle.tableContainer} ag-theme-alpine`
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals